commands/config/init.go

```diff
@@ -65,7 +65,14 @@ func runInitCommand(cmd *cobra.Command, args []string) {
 	if filepath == "" {
 		filepath = commands.Config.ConfigFile.String()
 	}
-	err := commands.Config.SaveToYAML(filepath)
+
+	err := os.MkdirAll(commands.Config.ConfigFile.Parent().String(), 0766)
```

Comment:
There should be a `MkdirAll` method directly from the `*paths.Path` object returned from `Parent()`.